serde-oracle-jdbc-json/src/test/groovy/io/micronaut/serde/oracle/jdbc/json/OracleJdbcJsonBinaryBasicSerdeSpec.groovy

```diff
@@ -0,0 +1,50 @@
+package io.micronaut.serde.oracle.jdbc.json
+
+import io.micronaut.core.type.Argument
+import io.micronaut.json.JsonMapper
+import io.micronaut.serde.AbstractBasicSerdeSpec
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import jakarta.inject.Inject
+import oracle.sql.json.OracleJsonFactory
+
+import java.nio.charset.StandardCharsets
+
+@MicronautTest
+class OracleJdbcJsonBinaryBasicSerdeSpec extends AbstractBasicSerdeSpec {
```

Comment:
Could you add more tests mirroring some of the tests in https://github.com/micronaut-projects/micronaut-serialization/tree/master/serde-bson/src/test/groovy/io/micronaut/serde/bson ?